tests/main/netplan-apply/task.yaml

```diff
@@ -0,0 +1,73 @@
+summary: Ensure that netplan apply works with network-setup-control
+
+details: |
+    Netplan apply is used to apply network configuration to the system
+
+# run on all classic ubuntu LTS and current dev systems 16.04+
```

Comment:
Can we inverse the selection, is there a place where it does not work?

Reply:
It does not work on ubuntu 14.04 and all non-ubuntu systems, but I wanted to avoid having to enumerate all non-ubuntu systems in case another distro gets added because then this test will fail there and someone will have to go and add that distro to the blacklist here.

Reply:
As such, I think it's preferable to just enumerate the systems it does work on, but if you feel strongly about it I can invert the selection

Reply:
FYI, I also felt inverting it made sense, but will defer to others.